zeth.txt

```diff
@@ -0,0 +1,14 @@
+[Section]
+Name = Zeth
+Version = 0.0.1
+License = Freeware
+Description = Zeth Visual Style for ReactOS
+Size = 116 KiB
+Category = 15
+URLDownload = https://github.com/Mas4hmad/ms-visual-styles/raw/master/ZETH.msstyles
```

Comment:
Does ReactOS install it correctly like this?

Reply:
that's not installer, just copy it manually into theme folder.

Reply:
In that case it would be a good idea to either create an installer, or add instructions in the description?

Reply:
for my first step, i would prefer add instruction. instead, create an installer is less possible regarding my lack of C language, i will make 'create an installer' if i ready.

Reply:
I usual make installer via 7zip SFX Builder.
https://sourceforge.net/projects/s-zipsfxbuilder/
You can open my sfx (*.exe) in it and learn how to make own.
https://github.com/petr-akhlamov/ros-repo/raw/master/lunabeta.exe

Reply:
All those installers ask the user where to install (and provide a correct default),
it's better to use f.e. winrar that allows to specify the target directory....

Reply:
thanks for the hint, will do update.
thank you so much before